schemas/session-replay/_common-shape-wireframe-schema.json

```diff
@@ -0,0 +1,58 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "$id": "_common-shape-wireframe-schema.json",
+  "title": "CommonShapeWireframe",
+  "type": "object",
+  "description": "Schema of common properties for ShapeWireframe events type and all its sub - types.",
+  "allOf": [
+    {
+      "$ref": "_common-wireframe-schema.json"
+    },
+    {
+      "properties": {
+        "shapeStyle": {
+          "type": "object",
+          "description": "The style of this wireframe.",
+          "properties": {
+            "fillColor": {
+              "type": "string",
+              "description": "The filling color for this wireframe as a String hexadecimal. Follows the #RRGGBBAA color format, default value should be #FFFFFF00.",
+              "readOnly": true
+            }
+          },
+          "opacity": {
+            "type": "number",
+            "description": "The opacity of this wireframe. Takes values from 0 to 1, default value should be 1.",
```

Comment:
In theory the opacity is defined by the alpha component of the `shapeStyle#fillColor`, so we are kind of duplicating info. Or is it meant to be applied on top what if defined in the `shapeStyle#fillColor`? Say we have 0.5 here and 0.5 in the `shapeStyle#fillColor`, resulting alpha will be 0.25 or 0.5?

If it is a duplication, then it is ok for me, because a) it is not a lot of data; b) has a better clarity. Alternatively we can just keep rgb and alpha in the different fields. If it is an additional alpha, then I think we should add a notice to the description.